components/table/demo/virtual-list.md

```diff
@@ -103,7 +103,7 @@ function VirtualTable(props) {
     >
       <Table
         {...props}
-        className={classNames(className, 'virtual-table')}
```

Comment:
We should remove `import classNames from 'classnames';`.

Reply:
We can’t, line 86 uses that.

Reply:
yes, i also know it

Reply:
i think just remove the unnecessary classname